templates/campaign_results.html

```diff
@@ -13,8 +13,13 @@
                 </li>
                 <li><a href="/landing_pages">Landing Pages</a>
                 </li>
+                <li><a href="/sending_profiles">Sending Profiles</a>
+                </li>
                 <li><a href="/settings">Settings</a>
                 </li>
+                <li><hr></li>
+                <li><a href="/gophish_user_guide.pdf">User Guide</a>
```

Comment:
Eventually, I'm probably going to move these to a generic "Documentation" option with all the docs as links.